### PR TITLE
FIX: Remove existing release notes when applying Catalyst patches.

### PR DESCRIPTION
**Pull request by marcosrdz**
Builds would get release notes from old commits rather than the latest ones

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/fkbf1j8rnufy4j5z6139ueavd0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7478)

File: scripts/maccatalystpatches/applypatchesformaccatalyst.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
+echo "Removing existing release notes"
+rm release-notes.txt release-notes.json
```
Comment (Overtorment) [inline scripts/maccatalystpatches/applypatchesformaccatalyst.sh:2]: how is this supposed to work?
those files should regenerate upon instal

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: scripts/maccatalystpatches/applypatchesformaccatalyst.sh (hunk 1 above)
Comment (marcosrdz) [reply scripts/maccatalystpatches/applypatchesformaccatalyst.sh:2]: Those files dont get over written when npm install runs so one could end up deploying release note files that are inaccurate. Such as from WIP branches

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**